galette/templates/default/pages/contributions_list.html.twig

```diff
@@ -144,57 +204,61 @@
                     {{ _T("Clear filter") }}
                 </button>
             </div>
+    {% if preferences.pref_use_paid_flag %}
         </div>
+    {% endif %}
     </div>
-    {% if member is defined and mode != 'ajax' %}
-        {% if login.isAdmin() or login.isStaff() or member.canShow(login) %}
-            <div class="ui compact vertically fitted segment">
-            <div class="ui horizontal list">
-            <span class="ui primary ribbon label">
-            {% set member_logged_in_as = member.name ~ " " ~ member.surname ~ " (" ~ member.login ~ ")" %}
-            {% if login.isAdmin() or login.isStaff() or member.hasChildren() or (member.hasParent() and (member_logged_in_as != login.loggedInAs(true))) %}
-                <a
-                    href="{{ url_for("contributions", {"type": "contributions", "option": "member", "value": "all"}) }}"
-                >
-                    <i class="icon times tooltip" aria-hidden="true"></i>
-                    <span class="ui special popup">
-                        {% if member.hasChildren() or (member.hasParent() and (member_logged_in_as != login.loggedInAs(true))) %}
-                            {{ _T("Show all your contributions") }}
-                        {% else %}
-                            {{ _T("Show all members contributions") }}
+    {% if mode != 'ajax' and member is defined and (login.isAdmin() or login.isStaff() or member.canShow(login)) %}
+        <div class="ui horizontal wrapping segments member-filtered">
+            <div class="ui compact segment">
+                <div class="ui horizontal list">
+                    <span class="ui primary ribbon label">
+                        {% set member_logged_in_as = member.name ~ " " ~ member.surname ~ " (" ~ member.login ~ ")" %}
+                        {% if login.isAdmin() or login.isStaff() or login.isGroupManager() or member.hasChildren() or (member.hasParent() and (member_logged_in_as != login.loggedInAs(true))) %}
```

Comment:
Hum... Link should be limited to groups managers if `pref_bool_groupsmanagers_see_contributions` is enabled

Reply:
```suggestion
                        {% if login.isAdmin() or login.isStaff() or login.isGroupManager() and preferences.pref_bool_groupsmanagers_see_contributions member.hasChildren() or (member.hasParent() and (member_logged_in_as != login.loggedInAs(true))) %}
```

Reply:
> Hum... Link should be limited to groups managers if `pref_bool_groupsmanagers_see_contributions` is enabled

I have to look back at this one to be sure, but I don't think so.

Adding this restriction, if I remember well, will prevent group managers that have children to list their children's contributions when clicking this link to close the toolbar.

Reply:
You're right !